frontend/src/features/admin-form/settings/components/EndPageSettingsSection/EndPageSettingsInput.tsx

```diff
@@ -0,0 +1,93 @@
+import { useCallback } from 'react'
+import { useForm } from 'react-hook-form'
+import { Flex, FormControl, Stack } from '@chakra-ui/react'
+import validator from 'validator'
+
+import { FormEndPage } from '~shared/types'
+
+import Button from '~components/Button'
+import FormErrorMessage from '~components/FormControl/FormErrorMessage'
+import FormLabel from '~components/FormControl/FormLabel'
+import Input from '~components/Input'
+import Textarea from '~components/Textarea'
+
+import { useMutateFormPage } from '~features/admin-form/common/mutations'
+
+interface EndPageSettingsInputProps {
+  endPage: FormEndPage
+}
+
+export const EndPageSettingsInput = ({
+  endPage,
+}: EndPageSettingsInputProps): JSX.Element => {
+  const { mutateFormEndPage } = useMutateFormPage()
+
+  const buttonLinkValidation = useCallback(
+    (url) =>
+      !url ||
+      validator.isURL(url, {
+        protocols: ['https'],
+        require_protocol: true,
+      }) ||
+      'Please enter a valid URL (starting with https://)',
+    [],
+  )
+
+  const {
+    register,
+    formState: { errors },
+    handleSubmit,
+  } = useForm<FormEndPage>({
+    mode: 'onChange',
+    defaultValues: endPage,
+  })
+
+  const handleUpdateEndPage = handleSubmit((endPage) =>
+    mutateFormEndPage.mutate(endPage),
+  )
+
+  return (
+    <Stack gap="2rem" paddingTop="2.5rem">
+      <FormControl isInvalid={!!errors.title}>
+        <FormLabel isRequired>Title</FormLabel>
+        <Input
+          {...register('title', {
+            required: 'Title is required for the Thank You page',
+          })}
+        />
+        <FormErrorMessage>{errors.title?.message}</FormErrorMessage>
+      </FormControl>
+      <FormControl isInvalid={!!errors.paragraph}>
+        <FormLabel isRequired>Follow-up instructions</FormLabel>
```

Comment:
looking at the current behavior, follow up instructions are not required. seems like only the title is required!